Celeste.Mod.mm/Mod/UI/TextMenuExt.cs

```diff
@@ -1480,6 +1481,7 @@ bool initiallyVisible
 
         public class TextBox : TextMenu.Item {
             private static readonly float DEFAULT_TEXT_SCALE = 1.10f;
+            private static readonly FieldInfo mInputVirtualInputsField = typeof(MInput).GetField("VirtualInputs", BindingFlags.NonPublic | BindingFlags.Static);
```

Comment:
`VirtualInputs` is `internal`, and since the code becomes part of Celeste when installed, you don't need reflection to access it ^^ you can access `patch_MInput.VirtualInputs` instead, which should be transformed into the real MInput on install

Reply:
Done.